BUILD

```
@@ -0,0 +1,31 @@
+load(
+    "@rules_cc//cc:defs.bzl",
+    "cc_library"
+)
+load(
+    "@build_bazel_rules_swift//swift:swift.bzl",
+    "swift_library"
+)
+
+cc_library(
+    name = "CYaml",
+    srcs = glob(["Sources/CYaml/src/*.c", "Sources/CYaml/src/*.h"]),
+    hdrs = ["Sources/CYaml/include/yaml.h"],
+    includes = ["Sources/CYaml/include"],
+    visibility = [
+        "//visibility:public",
```

```suggestion
        "//Tests:__subpackages__",
```


probably still only want this?